gutenTAG/visualization.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import os
+import pathlib
+
+PROJECT_DIR = pathlib.Path().resolve()
+sys.path.append(str(PROJECT_DIR))
+
+import pandas as pd
+import plotly.graph_objs as go
+
+# Define the path to the directory containing the timeseries files
+folder_name = 'generated-timeseries'
+folder_path = PROJECT_DIR.joinpath(folder_name)
+
+# Loop through all subdirectories and read the CSV files
+for subdir, dirs, files in os.walk(folder_path):
+    for file in files:
+        if file.endswith('.csv'):
+            file_path = os.path.join(subdir, file)
+            df = pd.read_csv(file_path)
+    
+            # Create a plot for each value column
+            fig = go.Figure()
+    
+            # Add the markers for anomalies
+            anomalies = df[df['is_anomaly'] == 1]
+            fig.add_trace(
+                go.Scatter(
+                    x=anomalies['timestamp'],
+                    y=anomalies['value-0'],
+                    mode='markers',
+                    marker=dict(color='red', size=5)
+                )
+            )
+            fig.add_trace(
+                go.Scatter(
+                    x=anomalies['timestamp'],
+                    y=anomalies['value-1'],
```

Comment:
This is probably covered but will this scale to when there are more than 2 values in `df` and `anomalies`?

Reply:
Probably won't scale. I've added a comment now to explain it's just a example